content/main_tester.sh

```diff
@@ -52,7 +52,7 @@ MEMLIMIT=$1
 shift
 
 # Effectively importing the testing function : validate_submission_output()
```

Comment:
This thing should disappear ideally.

Reply:
I'll delete the stale comments once you've tested the new code and are sure it works.